endpoints/openrtb2/video_auction_test.go

```diff
@@ -194,7 +196,9 @@ func mockDeps(t *testing.T, ex *mockExchangeVideo) *endpointDeps {
 		false,
 		[]byte{},
 		openrtb_ext.BidderMap,
-		nil}
+		empty_fetcher.EmptyFetcher{},
```

Comment:
Should we also be using a mock for category fetcher in these tests to make sure the categories are fetched and returned in the auction response as expected?